### PR TITLE
#3 standardize mailchimp endpoint

### LINKED ISSUE #3: Standardize MailChimp endpoint
This task is set to standardize the MailChimp endpoint.

Remember to add two or three important business flow steps.

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull request for the topic https://github.com/slingr-stack/mailchimp-endpoint/issues/3 created by @agreggio-slingr

## What does it do?
This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and step flows.

## How to test it?
Make calls from the platform for example using the console using the information provided in the documentation.

## Implementation notes
Create new version of the endpoint, it is not compatible with previous versions.

## Implementation notes
Create a new tags changing the major version and deploy to the platform.

### CODE REVIEW

**Comment by sebasmoyano** [comment]
#approve

File: endpoint.json [hunk 1]
```diff
@@ -68,6 +68,16 @@
     "mailchimp-fn-helpers.js",
```
Comment (pasaperez-work) [inline endpoint.json]: Remove unnecessary calls, or the builder will fail.

File: endpoint.json
Comment (pasaperez-work) [inline endpoint.json:1]: The functions:
_convertDateToTimestamp_convertDateToTimestamp ... _getResourceById
they are deprecated according to the new functions.js ... remove from the descriptor and the support in the main file: "src/main/java/io/slingr/endpoints/mailchimp/" .
Standardize the main file to leave http calls (through the framework) or as little custom code as possible for migration.

**Review by pasaperez-work — CHANGES_REQUESTED** [review]
Review Readme for changes in helpers, copy documentation generated from flows step, review consistency.

File: flowSteps/apiCallMailchimp/step.json [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+    {
+      "label": "Event",
+      "name": "events",
+      "type": "dropDown",
+      "description": "Select event",
+      "multiplicity": "one",
+      "options": {
+        "possibleValues": [
+          {
+            "label": "File Downloaded",
```
Comment (pasaperez-work) [inline flowSteps/apiCallMailchimp/step.json]: Could this field be a switcher (boolean)?

**Review by pasaperez-work — APPROVED** [review]

**Comment by pasaperez-work** [comment]
#approve

**Pull request merged**